source/BoardingPanel.cpp

```diff
@@ -59,9 +59,9 @@ namespace {
 
 
 // Constructor.
-BoardingPanel::BoardingPanel(PlayerInfo &player, const shared_ptr<Ship> &victim)
+BoardingPanel::BoardingPanel(PlayerInfo &player, const shared_ptr<Ship> &victim, bool overrideCapture)
```

Comment:
~~you don't like looping the missions in BoardingPanel do you~~

Reply:
What?

Reply:
The HailPanel thing I mentioned; it saves you from passing `overrideBoarding` everywhere because you would just check it in BoardingPanel itself.

Reply:
> I have to add a ship pointer to all mission
are you sure? can't you just check if the mission has overrideBoarding?